udata_hydra/crawl.py

```diff
@@ -227,19 +234,26 @@ async def check_url(row, session, sleep=0, method="get"):
                 }
             )
             return STATUS_OK
+    except asyncio.exceptions.TimeoutError:
+        await update_check_and_catalog(
+            {
+                "resource_id": row["resource_id"],
+                "url": row["url"],
+                "domain": domain,
+                "timeout": True,
+            }
+        )
+        return STATUS_TIMEOUT
     # TODO: debug AssertionError, should be caught in DB now
     # File "[...]aiohttp/connector.py", line 991, in _create_direct_connection
     # assert port is not None
     # UnicodeError: encoding with 'idna' codec failed (UnicodeError: label too long)
     # eg http://%20Localisation%20des%20acc%C3%A8s%20des%20offices%20de%20tourisme
-    except (
```

Comment:
Don't we want to keep these exhaustive exceptions, dealing with timeout separately? See also: https://github.com/etalab/udata-hydra/issues/17

Reply:
Indeed, I had not seen Alexandre's comment